depinject/README.md

```diff
@@ -137,17 +154,18 @@ In the above situation registering a binding for a given interface binding may l
 
 ```go
 depinject.Inject(
-  depinject.Configs(
-    depinject.BindInterface(
-      "duck.Duck",
-      "duck.Mallard"),
-     depinject.Provide(
-       func() Mallard { return Mallard{} },
-       func() Canvasback { return Canvasback{} },
-       func(duck Duck) APond {
-         return Pond{Duck: duck}
-      })),
-   &pond)
+	depinject.Configs(
+		depinject.BindInterface(
+			"duck/duck.Duck",
+			"duck/duck.Mallard",
```

Comment:
As mentioned in the PR body, when running this code in a go module called "duck", using `"duck.Duck"` and `"duck.Mallard"` as `BindInterface` arguement isn't working. I have to replace them with  `"duck/duck.Duck"` and `"duck/duck.Mallard"`. Not sure if it's because of my module name or something else.

Reply:
ah OK already merged, @julienrbrt is too fast :)

Reply:
Yeah it should be full path, so this is a good change yes!